parquet/src/main/java/org/apache/iceberg/parquet/ParquetTypeVisitor.java

```diff
@@ -29,9 +29,8 @@
 import org.apache.parquet.schema.PrimitiveType;
 import org.apache.parquet.schema.Type;
 
-import static org.apache.parquet.schema.Type.Repetition.REPEATED;
-
 public class ParquetTypeVisitor<T> {
+  @SuppressWarnings({"checkstyle:IllegalType", "checkstyle:VisibilityModifier"})
```

Comment:
I think other places changed to use `Deque` instead of suppressing the warning.

Reply:
Sweet, thanks